data/types/base/device/alarm/alarm.go

```diff
@@ -54,7 +61,22 @@ func (a *Alarm) Parse(parser data.ObjectParser) error {
 	}
 
 	a.AlarmType = parser.ParseString("alarmType")
-	a.Status = parser.ParseString("status")
+
+	// TODO: This is a bit hacky to ensure we only parse true status data. Is there a better way?
```

Comment:
Actually, this is my hacky bit.  ;)  It is fine for now and is technically correct, but just doesn't feel right. I'll look at it again later and see if I can come up with something a bit better.